bpf/tests/prog_test/prog_test.go

```diff
@@ -17,18 +17,18 @@ import (
 	"reflect"
 	"testing"
 
+	"github.com/cilium/ebpf"
+	"github.com/cilium/ebpf/perf"
+	"github.com/google/gopacket"
+	"github.com/google/gopacket/layers"
+	log "github.com/sirupsen/logrus"
+
 	"github.com/cilium/cilium/pkg/bpf"
 	"github.com/cilium/cilium/pkg/byteorder"
 	"github.com/cilium/cilium/pkg/maps/ctmap"
 	"github.com/cilium/cilium/pkg/monitor"
 	"github.com/cilium/cilium/pkg/tuple"
 	"github.com/cilium/cilium/pkg/types"
-	"github.com/cilium/ebpf"
-	"github.com/cilium/ebpf/perf"
-
-	"github.com/google/gopacket"
-	"github.com/google/gopacket/layers"
-	log "github.com/sirupsen/logrus"
 	"golang.org/x/sys/unix"
```

Comment:
I think this should be part of the second import group (3rd party modules) as well. Not sure whether this is a bug in `goimports`? It looks like it got ordered correctly in other files.

Reply:
Thanks, fixed.

For info, `goimports` (the command, not the linter) seems to ignore blocks of imports after the third block, which meant that quite a lot of manual fixing up was needed after running `goimports -w`.